recipes/metabat/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "metabat" %}
+{% set version = "0.32.4" %}
+{% set sha256linux = "6db36f8d43e4e7d246c4f5028979952d8f29ed5fe69ae8175fb4f2be98cb9a22" %}
+{% set sha256osx = "d17fcee3891f2f478379edff30c5eee98dd8d76b2d876859f12087828abb142a" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  # Linux binary
+  url: https://bitbucket.org/berkeleylab/{{ name }}/downloads/{{ name }}-static-binary-linux-x64_v{{ version }}.tar.gz  # [linux]
+  sha256linux: {{ sha256linux }}  # [linux]
+
+  # OSX binary
+  url: https://bitbucket.org/berkeleylab/{{ name }}/downloads/{{ name }}-static-binary-osx-elcapitan-x64_v{{ version }}.tar.gz  # [osx]
+  sha256: {{ sha256osx }}  # [osx]
```

Comment:
Compiling from sources requires the Samtools sources and Boost.
The default makefile automatically download and build samtools (1.2) on its own. This won't be maintainable for long term. A proper solution would be to use samtools as a independent lib but this would involve major changes in the software architecture.
Boost is searched in folder like `/usr/lib`, but a simple patch could fix that.
Regarding these, I found that using static binaries is much more simple and maintainable (but depends on software authors).